projects/plugins/crm/includes/ZeroBSCRM.InvoiceBuilder.php

```diff
@@ -1086,15 +1082,21 @@ function zeroBSCRM_invoicing_generateInvoiceHTML( $invoice_id = -1, $template =
 			// ignore if status_bool (non-completed status)
 			$partial['status_bool'] = (int) $partial['status_bool'];
 			if ( isset( $partial ) && $partial['status_bool'] == 1 ) { // phpcs:ignore Universal.Operators.StrictComparisons.LooseEqual
+				// v3.0+ has + or - partials. Account for that.
+				switch ( $partial['type'] ) {
 
-				// v3.0+ has + or - partials. Account for that:
-				if ( $partial['type_accounting'] === 'credit' ) {
-					// credit note, or refund
-					$balance = $balance + $partial['total'];
-				} else {
-					// assume debit
-					$balance = $balance - $partial['total'];
-				}
+					case __( 'Sale', 'zero-bs-crm' ):
```

Comment:
In these cases we're switching from checking the 'type_accounting', to checking for the 'type' (and the translated string too). It was noted here that isn't ideal - https://github.com/Automattic/zero-bs-crm/pull/2129/files#r875102950 - but that this is a bigger issue throughout. In this case, we're being consistent with the approach in `ZeroBSCRM.DAL3.Obj.Invoices.php`.